src/components/ProblemEditorList/ProblemEditorList.i18n/en.json

```diff
@@ -0,0 +1,6 @@
+{
+    "Problem Editors": "",
+    "Add Problem Editor role": "",
+    "Take the role of problem editor from": "",
+    "Add problem editor role": ""
```

Comment:
why this key used twice(on 3 and 5 string)? and only once in `ru.json`?

Reply:
Because I need to delete an unused key